### ISSUE TITLE
twistlock integration

### ISSUE THREAD

**Issue by kkhamruddin-pk** [goharbor/harbor#6372]
Does Vmware Harbor supports twistlock integration?

**Comment by clouderati** [comment]
No, not at the moment. We'll be discussing modularizing Harbor so that integration with 3rd party platforms is an attainable goal. Let me know if you're interested in joining and how I can reach you. Thanks!

**Comment by michmike** [comment]
hi  @kkhamruddin-pk with the 1.10 release of Harbor we will introduce pluggable scanners, with the first 2 scanners coming from Aqua and Anchore. you should check it out. we also have demos of these in our community meetings. (https://www.youtube.com/channel/UCSxaozHKrX3F0UnZeYe5Itg/playlists) watch for the 10/09/2019 presentation that will be uploaded in a couple of days

**Comment by shahidv3** [comment]
Hi 

Does Harbor officially supports Twistlock ( Prisma cloud compute ) 

I tried adding Twistlock as scanner in Harbor but it throws error as Internal error and connection reset by Peer as ICMP is blocked on harbor server in my organisation 

please let me know of any alternatives 

thanks

**Comment by drduker** [comment]
Curious on the status of this, thanks!
